xml/deployment_ganesha.xml

```diff
@@ -408,6 +408,103 @@ Full list of resources:
    </procedure>
   </sect2>
 
+  <sect2 xml:id="setup-portblock-resource">
+    <title>Setting Up PortBlock Resource</title>
+    <para>When a service goes down, the TCP connection that is in use by
+      nfs-&ganesha; is required to be closed otherwise it
+      continues to run until a system-specific timeout occurs. This timeout
+      can take upwards of 3 minutes.</para>
+    <para>To shorten the timeout time, the TCP connection needs to be reset.
+      We recommend configuring <literal>portblock</literal> to reset stale TCP
+      connections.</para>
+    <para>You can choose to use portblock with or without the
+      <literal>tickle_dir</literal> parameters that could unblock and
+      reconnect clients to the new service faster. We recommend to
+      have <literal>tickle_dir</literal> as the shared CephFS mount
+      between two HA nodes (where nfs-&ganesha; services are running).</para>
+      <procedure>
+        <note>
+          <para>Configuring the following resource is optional.</para>
+        </note>
+        <step>
+          <para>On &si.earth;, run the crm shell to execute the following
+            commands to configure the resource for NFS Ganesha daemons:</para>
+<screen>&prompt.earth;<command>crm</command> configure</screen>
+        </step>
+        <step>
+          <para>Configure the <literal>block</literal> action for
+          <literal>portblock</literal> and omit the <literal>tickle_dir</literal>
+          option if you haven't configured a shared directory:</para>
+<screen><prompt xmlns="http://docbook.org/ns/docbook">crm(live)configure# </prompt> primitive nfs-ganesha-block ocf:portblock protocol=tcp portno=2049 action=block ip=192.168.2.1 opmonitor depth="0" timeout="10" interval="10" tickle_dir="/tmp/ganesha/tickle/"</screen>
```

Comment:
This screen line is probably longer > 80 characters and will break in PDF output

Reply:
`opmonitor` should be `op monitor`